modules/editor/assets/stylesheets/main.scss

```diff
@@ -1,12 +1,19 @@
+@import '../../inserter/style';
+
 .gutenberg {
 	background: #fff;
 	margin: -20px 0 0 -20px;
 	padding: 60px;
+
+	* {
+		box-sizing: border-box;
```

Comment:
I agree CSS modules is really nice. Maybe, we could consider it once we settle the other moving parts.